src/analytics/subscriber_notification.rs

```diff
@@ -30,7 +30,7 @@ pub struct SubscriberNotification {
     /// Hash of the CAIP-10 account of the subscriber
     pub account_hash: String,
     /// The notification type ID
-    pub notification_type: Arc<str>,
+    pub notification_type: String,
```

Comment:
Because the notification type was internally changed from `Arc<str>` to a UUID, here it's creating a new String representation for it via `.to_string()` and converting it to an `Arc<str>` doesn't provide any value